docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth to mention that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
+
+[0]: https://gnupg.org
+[1]: https://quay.io
+
+From a technical perspective, we can utilize Podman to sign the image before
+pushing it into a remote registry. After that, all systems running Podman have
+to be configured to retrieve the sign key from a remote server, which can be any
```

Comment:
sign key -> signed key.   Although maybe signature key?  Others have thoughts?

Reply:
Decided to go for `signature key`

Reply:
Is this talking about a _key_ at all? The talk about a remote server suggests this is about the sigstore and signatures.

Reply:
I changed it to `signatures` to be more explicit. It's just referring the remote server contents.